cmd/src/campaigns_common.go

```diff
@@ -236,16 +246,11 @@ func campaignsExecute(ctx context.Context, out *output.Output, svc *campaigns.Se
 		KeepLogs:    flags.keepLogs,
 		Timeout:     flags.timeout,
 		TempDir:     flags.tempDir,
-	}
-	if flags.parallelism <= 0 {
-		opts.Parallelism = runtime.GOMAXPROCS(0)
-	} else {
-		opts.Parallelism = flags.parallelism
+		Parallelism: flags.parallelism,
 	}
 
-	executor := svc.NewExecutor(opts)
```

Comment:
I plan of changing how `*Service` and `*Executor` are instantiated in this function and this was a first step before I realised that doing it would make the PR here super noisy.